docs/installing.rst

```diff
@@ -11,25 +11,27 @@ Installing the Atomic Developer Bundle
 1. Install a Virtualization Provider
```

Comment:
Could we fix double bullets issues as like below if we click on view button:

![double_bullets](https://cloud.githubusercontent.com/assets/1132451/15928711/22fb96c4-2e66-11e6-84ac-dd61a62cf65b.png)